main.go

```diff
@@ -89,6 +89,9 @@ func main() {
 	username := authTokens[0]
 	password := authTokens[1]
 	registry := registryURL.Host
+
+	shortRepoName := vargs.Repo
```

Comment:
It's needed because of https://github.com/gregory90/drone-ecr/blob/master/main.go#L99